src/commands/migration_tasks.py

```diff
@@ -1421,9 +1435,25 @@ def list_fund_rows(kiid_session: Session):
         Returns: rows from the kiid funds table
         """
         risk_query = "SELECT TOP 1 risk_level FROM TextContent WHERE fund_name = NAME_KIID ORDER BY modification_time"
-        statement = f"SELECT ID, URL_FI, URL_SV, URL_EN, ({risk_query}) as RISK_LEVEL FROM FUND"
+        statement = f"SELECT ID, URL_FI, URL_SV, URL_EN, ({risk_query}) as RISK_LEVEL, FUND_TYPE FROM FUND"
         return kiid_session.execute(statement=statement)
 
+    @staticmethod
+    def get_fund_group(fund_type: str) -> str:
+        """
+        Returns fund group from fund type
+        Args:
+            fund_type: fund type
+
+        Returns:
+            fund group
+        """
+        fund_group = re.sub(r'(?=[A-Z])', '_', fund_type).upper()
+        if fund_group in ["PASSIVE", "ACTIVE", "BALANCED", "FIXED_INCOME", "DIMENSION", "SPILTAN"]:
```

Comment:
Lets have this list defined as a constant/enum somewhere(in a constants file?) out of these function.